src/index.scss

```diff
@@ -5,4 +5,5 @@
   padding: 0;
   box-sizing: border-box;
   font-family: 'Raleway', sans-serif;
-}
\ No newline at end of file
+}
+
```

Comment:
That's the exact kind of change I don't want to see anymore... Is anyone in your group working without Prettier/ESLint ?
If I see this again, I'll close PRs without discussion.